src/state/apis/nft/constants.ts

```diff
@@ -29,15 +29,21 @@ const NFT_NAME_BLACKLIST = [
   'ethstation',
   'jrnyclubnet',
   'jrnyclub.net',
+  '://',
 ]
 
+const TOKEN_TEXT_REGEX_BLACKLIST = [/:\/\//, /\.[a-zA-Z]{2,4}/, /ETH\.\.\./]
+
 // This escapes special characters we may encounter in NFTS, so we can add them to the blacklist
 // e.g "$9999+ free giveaway *limited time only*" would not work without it
 const nftNameBlacklistRegex = new RegExp(
   NFT_NAME_BLACKLIST.map(term => term.replace(/[.*+?^${}()|[\]\\]/g, '\\$&')).join('|'),
   'i',
 )
-export const isSpammyNftText = (nftText: string) => nftNameBlacklistRegex.test(nftText)
+export const isSpammyNftText = (nftText: string) =>
+  nftText === '' || nftNameBlacklistRegex.test(nftText)
```

Comment:
Could we just make `nftNameBlacklistRegex` check for empty strings, too?

Reply:
Agreed, though it's currently a lazy regex, so would have to make sure to test against the current interesection of `nftNameBlacklistRegex` built regex, and a `^$` regex for empty strings, vs. adding it to `NFT_NAME_BLACKLIST` which would filter all

Reply:
https://github.com/shapeshift/web/pull/5728/commits/22a353bf4dfff8360ece84701aea58be67070ad8